src/views/Player.vue

```diff
@@ -403,9 +415,16 @@ export default {
 
       player.on('play', event => {
           const instance = event.detail.plyr;
-          instance.fullscreen.enter()
-          if (!this.supportedBrowsers.includes(this.browser))
-              this.checkBrowserSupport();
+
+          if (!instance.fullscreen.active) instance.fullscreen.enter()
+
+          if (!this.supportedBrowsers.includes(this.browser)) {
+            this.checkBrowserSupport();
+          }
+          else {
+            this.hasVideoPlayed = true
+            this.isBrowserSupported = true
+          }
```

Comment:
Everytime play event was fired, it was trying to make it fullscreen, which gave exception sometimes.
Now it will only go to fullscreen if it's already not in fullscreen.

Also, if the current browser is supported, we'll skip the `checkBrowserSupport` check

Reply:
Default of `isBrowserSupported` is already `True` -> So, this is not needed right?
Also, we should be consistent in what we use for `hasVideoPlayed`. In `checkBrowserSupport`, it is a binary int, here it is a `bool`.

Reply:
Default of `isBrowserSupported` is already `True` -> So, this is not needed right?
Also, we should be consistent in what we use for `hasVideoPlayed`. In `checkBrowserSupport`, it is a binary int, here it is a bool.

Reply:
Correct and correct. Fixing and pushing